src/semantic_digital_twin/world_description/connections.py

```diff
@@ -144,6 +145,11 @@ class ActiveConnection1DOF(ActiveConnection, ABC):
     Name of a Degree of freedom to control movement along the axis.
     """
 
+    prop: JointProperty = field(default_factory=JointProperty)
+    """
+    Dynamic properties of the joint.
+    """
```

Comment:
i'm pretty sure this should belong to degrees of freedom instead

Reply:
I thought about that, but you defined a degree of freedom can be an element of a quaternion, which I can't apply the property there